### PR TITLE
Completed html-forms

### PR DESCRIPTION
**Pull request by amorabryan**
image

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      <div class="html_forms">
+        <label for="question">Why is coding cool?</label>
+      </div>
+
+      <div class="html_forms">
+        <select id="reason" name="user_opinion">
+          <option>Select an option</option>
+          <option>Because it's interesting</option>
+          <option>Because it's cool</option>
+          <option>Because it's powerful</option>
```
Comment (kurtyazdizadeh) [inline html-forms/index.html]: make sure these `option` tags have a `value` attribute, this is what will get sent in the form submission.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      <input type="radio" name="Language" id="Language_3" value="JavaScript" />
+      <label for="Language_3">JavaScript</label>
+
+      <div class="html_forms">
+        Would you like to sign up for more information?
+      </div>
+
+      <div class="html_forms">
+        <input type="checkbox" id="sign" name="signup">
+        <label for="scales">Sign Me Up!</label>
```
Comment (kurtyazdizadeh) [inline html-forms/index.html]: I think you meant for this `for="sign"` to be associated with the checkbox?

**Review by kurtyazdizadeh — CHANGES_REQUESTED** [review]
Just a couple changes, see comments

File: html-forms/index.html (hunk 2 above)
Comment (amorabryan) [reply html-forms/index.html]: Oh yes I was looking at the example from MDN and I went on autopilot, thank you for the correction!

**Review by kurtyazdizadeh — APPROVED** [review]
Approved 👍

**Pull request merged**